service/build.gradle

```diff
@@ -136,6 +136,10 @@ dependencyCheck {
   }
 }
 
+// Excludes sonar cloud testing to avoid duplication of code warning
+def sonarExclusions = ['**/model/OldChild.java',
```

Comment:
is this still required?

Reply:
Creating a variable for it no, but it's a more maintainable if we need to add more exclusions?

Reply:
this is a temp fix, so I would just define below, then when removing, it's only two lines and they are next to each other to remove.

Reply:
maybe because it's temp this isn't important though 🤷‍♂